src/Veldrid/TextureViewDescription.cs

```diff
@@ -41,9 +45,11 @@ public struct TextureViewDescription : IEquatable<TextureViewDescription>
         /// </summary>
         /// <param name="target">The desired target <see cref="Texture"/>. This <see cref="Texture"/> must have been created
         /// with the <see cref="TextureUsage.Sampled"/> usage flag.</param>
-        public TextureViewDescription(Texture target)
+        /// <param name="viewType">The view type to create or <see cref="TextureViewType.Undefined"/> to detect from texture.</param>
+        public TextureViewDescription(Texture target, TextureViewType viewType = TextureViewType.Undefined)
```

Comment:
All of these constructor changes need to be reverted, because adding a new parameter (even if it has a default value) is not a backwards-compatible change for the ABI. Old code would have to be recompiled to work with these new signatures.

In order to add new a new field like this, you should add one single new constructor that accepts all of the other fields, plus the new one. This is generally why the set of constructors for Veldrid types looks the way it does -- new values have been added over time and new constructors with them.

Reply:
Done!